.github/workflows/tests.yml

```diff
@@ -274,6 +274,10 @@ jobs:
         if: matrix.build_type == 'system'
         run: script/assemble_ember_build.rb
 
+      - name: Install playwright
+        if: matrix.build_type == 'system'
+        run: ./node_modules/.bin/playwright install
```

Comment:
Does this command need to be ran locally as well? I don't see it being mentioned in the developer docs.

Reply:
Not blocking but we should move the installation of `playwright` into https://github.com/discourse/discourse_docker/blob/main/image/discourse_test/Dockerfile once this PR has been merged. 

Do let me know if you need help with this change.

Reply:
> Does this command need to be ran locally as well? I don't see it being mentioned in the developer docs.

Yes Im not sure, this is weird, it still run for me locally even if I run `playwright uninstall` I wonder if there's some cache going on here.

Reply:
ok I think I got this right, Im not sure about the docker file, we use chrome for the js tests too no? and playwright is using the new headless here. Not sure what is the best setup in the end...